web/sites/default/config/core.entity_view_display.paragraph.form_app_button.default.yml

```diff
@@ -0,0 +1,28 @@
+uuid: c3062357-89a4-4192-adef-5657082964f1
+langcode: en
+status: true
+dependencies:
+  config:
+    - field.field.paragraph.form_app_button.field_form_button
+    - paragraphs.paragraphs_type.form_app_button
+  module:
+    - link
+id: paragraph.form_app_button.default
+targetEntityType: paragraph
+bundle: form_app_button
+mode: default
+content:
+  field_form_button:
+    weight: 0
+    label: hidden
+    settings:
+      trim_length: 80
+      url_only: false
```

Comment:
There is a setting "Allow link text", please make it "Required" instead of "Optional"